cmd/posix-list-dir-nix.go

```diff
@@ -168,3 +160,44 @@ func readDirN(dirPath string, count int) (entries []string, err error) {
 	}
 	return
 }
+
+// getDiskUsage returns du information for the posix path, recursively.
+func getDiskUsage(diskPath string, buf []byte) (uint64, error) {
```

Comment:
Are we adding only the file sizes and not adding the directory metadata sizes?

Reply:
Directories do not exist on object storage @kannappanr they are basically 0 bytes, so prefixes can be simply ignored. These metadata are negligible and serve no purpose.  We are only interested in what we created such as objects and other metadata JSON files etc.